docs/_guides/minikube.md

```diff
@@ -78,7 +78,7 @@ kube-system   storage-provisioner           1/1       Running   0          42s
 ```
 $ brew install kubernetes-helm
 # If not using the tiller service account, simply run `helm init`
```

Comment:
Likely you should remove this comment as well. Or provide alternate paths for both.

Reply:
removed.  As long as you follow the happy path (i.e. kubeadm) you shouldn't need to worry about service accounts.